### PR TITLE
Add support for the "routing ospf static-neighbor" path

### PR DESCRIPTION
**Pull request by hansmi**
##### SUMMARY

Upstream documentation:
https://help.mikrotik.com/docs/display/ROS/OSPF#OSPF-StaticNeighbourconfiguration

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: changelogs/fragments/302-ospf-static-neighbor.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - add support for the ``routing ospf static-neighbor`` path in RouterOS 7 (https://github.com/ansible-collections/community.routeros/issues/302).
```
Comment (felixfontein) [inline changelogs/fragments/302-ospf-static-neighbor.yml:2]:
```suggestion
  - api_info, api_modify - add support for the ``routing ospf static-neighbor`` path in RouterOS 7 (https://github.com/ansible-collections/community.routeros/pull/302).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Just one small change :)

File: changelogs/fragments/302-ospf-static-neighbor.yml (hunk 1 above)
Comment (hansmi) [reply changelogs/fragments/302-ospf-static-neighbor.yml:2]: Thank you for noticing. Done, also in `300-add-ip-dhcp-server-matcher.yml` and `300-set-ip-dhcp-option-force-default.yml`.

Comment (felixfontein) [reply changelogs/fragments/302-ospf-static-neighbor.yml:2]: Thanks! You haven't changed it in this file though :)

Comment (hansmi) [reply changelogs/fragments/302-ospf-static-neighbor.yml]: Whops, that wasn't intentional of course. Done.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@hansmi thanks a lot!

**Pull request merged**